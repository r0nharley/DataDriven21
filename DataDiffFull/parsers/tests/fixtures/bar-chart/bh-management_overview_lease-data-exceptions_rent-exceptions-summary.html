<svg version="1.1" class="highcharts-root" style="font-family:&quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, Arial, Helvetica, sans-serif;font-size:12px;" xmlns="http://www.w3.org/2000/svg" width="1454" height="243">
    <desc>Created with Highstock 5.0.6</desc>
    <defs>
        <clipPath id="highcharts-6xri21s-4">
            <rect x="0" y="0" width="197" height="1266" fill="none"></rect>
        </clipPath>
        <clipPath id="highcharts-6xri21s-5">
            <rect x="0" y="0" width="197" height="1266" fill="none"></rect>
        </clipPath>
    </defs>
    <rect fill="#ffffff" class="highcharts-background" x="0" y="0" width="1454" height="243" rx="0" ry="0"></rect>
    <rect fill="none" class="highcharts-plot-background" x="178" y="10" width="1266" height="197"></rect>
    <g class="highcharts-grid highcharts-xaxis-grid ">
        <path fill="none" class="highcharts-grid-line" d="M 178 109.5 L 1444 109.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 178 207.5 L 1444 207.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 178 10.5 L 1444 10.5" opacity="1"></path>
    </g>
    <g class="highcharts-grid highcharts-yaxis-grid ">
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 177.5 10 L 177.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 240.5 10 L 240.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 304.5 10 L 304.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 367.5 10 L 367.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 430.5 10 L 430.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 494.5 10 L 494.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 557.5 10 L 557.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 620.5 10 L 620.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 683.5 10 L 683.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 747.5 10 L 747.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 810.5 10 L 810.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 873.5 10 L 873.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 937.5 10 L 937.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 1000.5 10 L 1000.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 1063.5 10 L 1063.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 1127.5 10 L 1127.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 1190.5 10 L 1190.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 1253.5 10 L 1253.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 1316.5 10 L 1316.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 1380.5 10 L 1380.5 207" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 1444.5 10 L 1444.5 207" opacity="1"></path>
    </g>
    <rect fill="none" class="highcharts-plot-border" x="178" y="10" width="1266" height="197"></rect>
    <g class="highcharts-axis highcharts-xaxis ">
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 178 109.5 L 168 109.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 178 207.5 L 168 207.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 178 9.5 L 168 9.5" opacity="1"></path>
        <path fill="none" class="highcharts-axis-line" stroke="#ccd6eb" stroke-width="1" d="M 177.5 10 L 177.5 207"></path>
    </g>
    <g class="highcharts-axis highcharts-yaxis ">
        <path fill="none" class="highcharts-axis-line" d="M 178 207 L 1444 207"></path>
    </g>
    <g class="highcharts-series-group">
        <g class="highcharts-series highcharts-series-0 highcharts-bar-series highcharts-color-undefined highcharts-tracker " transform="translate(1444,207) rotate(90) scale(-1,1) scale(1 1)" width="1266" height="197" clip-path="url(https://secure.rentlytics.com/bh-management/bi/overview-lease-data-exceptions/visualization#highcharts-6xri21s-5)">
            <rect x="150.5" y="63.5" width="24" height="1203" fill="#00cee6" class="highcharts-point " stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-0 highcharts-bar-series highcharts-color-undefined " transform="translate(1444,207) rotate(90) scale(-1,1) scale(1 1)" width="1266" height="197" clip-path="none"></g>
        <g class="highcharts-series highcharts-series-1 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(1444,207) rotate(90) scale(-1,1) scale(1 1)" width="1266" height="197" clip-path="url(https://secure.rentlytics.com/bh-management/bi/overview-lease-data-exceptions/visualization#highcharts-6xri21s-5)">
            <rect x="22.5" y="1228.5" width="24" height="38" fill="#9b9bd7" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-1 highcharts-bar-series highcharts-color-undefined " transform="translate(1444,207) rotate(90) scale(-1,1) scale(1 1)" width="1266" height="197" clip-path="none"></g>
    </g>
    <g class="highcharts-data-labels highcharts-series-0 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(178,10) scale(1 1)" opacity="1">
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(1203,23)"></g>
    </g>
    <g class="highcharts-data-labels highcharts-series-1 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(178,10) scale(1 1)" opacity="1">
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(38,151)"></g>
    </g>
    <g class="highcharts-axis-labels highcharts-xaxis-labels ">
        <text x="163" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:470px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="64" opacity="1">
            <tspan>Missing or $0 Actual Rent</tspan>
        </text>
        <text x="163" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:470px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="162" opacity="1">
            <tspan>Missing or $0 Market Rent</tspan>
        </text>
    </g>
    <g class="highcharts-axis-labels highcharts-yaxis-labels ">
        <text x="178" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">0</text>
        <text x="241.3" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">10</text>
        <text x="304.6" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">20</text>
        <text x="367.9" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">30</text>
        <text x="431.2" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">40</text>
        <text x="494.5" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">50</text>
        <text x="557.8" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">60</text>
        <text x="621.1" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">70</text>
        <text x="684.4" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">80</text>
        <text x="747.7" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">90</text>
        <text x="811" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">100</text>
        <text x="874.3" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">110</text>
        <text x="937.6" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">120</text>
        <text x="1000.9" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">130</text>
        <text x="1064.2" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">140</text>
        <text x="1127.5" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">150</text>
        <text x="1190.8" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">160</text>
        <text x="1254.1" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">170</text>
        <text x="1317.4" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">180</text>
        <text x="1380.7" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">190</text>
        <text x="1433.0703125" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="226" opacity="1">200</text>
    </g>
    <g class="highcharts-label highcharts-tooltip highcharts-color-undefined" style="cursor:default;pointer-events:none;white-space:nowrap;" transform="translate(1174,-9999)" opacity="0" visibility="visible">
        <path fill="rgba(255, 255, 255, 0)" class="highcharts-label-box highcharts-tooltip-box" d="M 0 0 L 201 0 C 201 0 201 0 201 0 L 201 37 207 43 201 49 201 102 C 201 102 201 102 201 102 L 0 102 C 0 102 0 102 0 102 L 0 0 C 0 0 0 0 0 0"></path>
    </g>
</svg>