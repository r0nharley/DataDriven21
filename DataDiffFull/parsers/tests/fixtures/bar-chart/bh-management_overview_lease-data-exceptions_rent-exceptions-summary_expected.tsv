Missing or $0 Actual Rent	Data	190.047393365
Missing or $0 Market Rent	Data	6.00315955766